modular-docs-manual/content/topics/using_text_fragments.adoc

```diff
@@ -0,0 +1,19 @@
+[id="using_text_fragments_{context}"]
+
+= Text Fragments
+
+A text fragment is a small section of text that is stored in an Asciidoc file. Text fragments contain content that is resused in multiple modules or assemblies, for example:
+
+* A step or series of steps in a procedure
+* A disclaimer, for example, for technology preview or beta releases
+
+NOTE: A text fragment is not a module. The Asciidoc file that contains the text fragment cannot contain any other information. A module cannot contain an include statement for another module.
```

Comment:
> A module cannot contain an include statement for another module.
Should this be: `A text fragment cannot contain an include statement for another module.`?

Reply:
Wow, that is a good point! I've just simplified the whole idea.